packages/entities/entities-plugins/docs/plugin-form.md

```diff
@@ -83,19 +103,67 @@ The base konnect or kongManger config.
 
 If showing the `Edit` type form, the ID of the plugin.
 
+#### `hideScopeSelection`
+
+- type: `Boolean`
+- required: `false`
+- default: `false`
+
+Manually toggle visibility of plugin scope selection control.
+
+#### `isCredential`
```

Comment:
nit: can we move away from prefixed prop names? `is*` and `has*`

At least for future as well